nni/retiarii/converter/graph_gen.py

```diff
@@ -654,8 +644,214 @@ def convert_module(self, script_module, module, module_name, ir_model):
 
         return ir_graph, {}
 
+    def convert_module(self, script_module, module, module_name, ir_model):
```

Comment:
why we need both `convert_module` and `_convert_module`?

Reply:
`_convert_module` is exposed to subclass to overwrite internal logic.